afqinsight/nn/tests/test_performance_py_tf.py

```diff
@@ -0,0 +1,272 @@
+import math
+
+import torch
+
+from afqinsight.nn.performance_utils import (
+    prep_pytorch_data,
+    prep_tensorflow_data,
+    test_pytorch_model,
+    test_tensorflow_model,
+)
+from afqinsight.nn.pt_models import (
+    blstm1_pt,
+    blstm2_pt,
+    cnn_lenet_pt,
+    cnn_resnet_pt,
+    cnn_vgg_pt,
+    lstm1_pt,
+    lstm1v0_pt,
+    lstm2_pt,
+    lstm_fcn_pt,
+    mlp4_pt,
+)
+from afqinsight.nn.tf_models import (
+    blstm1,
+    blstm2,
+    cnn_lenet,
+    cnn_resnet,
+    cnn_vgg,
+    lstm1,
+    lstm1v0,
+    lstm2,
+    lstm_fcn,
+    mlp4,
+)
+
+if torch.backends.mps.is_available():
+    # device = torch.device("mps")
+    device = torch.device("cpu")
+else:
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+train_dataset, X_test, X_train, y_test, val_dataset = prep_tensorflow_data()
+torch_dataset, train_loader, test_loader, val_loader = prep_pytorch_data()
+
+
+def test_mlp4():
```

Comment:
I would split these into two: pytorch and tensorflow separately. 

Even more revolutionary: make this two separate test files.